_database-integrations/mssql/google-cloud/v1/google-cloud-v1.md

```diff
@@ -0,0 +1,164 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/databases/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: "Google CloudSQL SQL Server (v1)"
+keywords: google-cloudsql-microsoft-sql-server, database integration, etl google-cloudsql-microsoft-sql-server, google-cloudsql-microsoft-sql-server etl
+permalink: /integrations/databases/google-cloudsql-microsoft-sql-server
+summary: "Connect and replicate data from your Google CloudSQL SQL Server database using Stitch's Microsoft SQL Server integration."
+
+show-in-menus: true
+key: "google-cloudsql-microsoft-sql-server-integration"
+
+# -------------------------- #
+#     Integration Details    #
+# -------------------------- #
+
+name: "google-cloudsql-microsoft-sql-server"
+display_name: "Google CloudSQL SQL Server"
+setup-name: "mssql"
+
+singer: true
+tap-name: "Microsoft SQL Server"
+repo-url: "https://github.com/singer-io/tap-mssql"
+
+this-version: "1"
+
+hosting-type: "google-cloudsql" ## amazon, microsoft, google, etc.
+
+driver: "7.2.1.jre8"
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+#feature-summary: |
+Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.driver | flatify | strip }}. [TODO]
```

Comment:
This needs to be commented out as well. This should've caused build errors locally.